perf/.gitignore

```diff
@@ -0,0 +1 @@
+pickle_path
```

Comment:
Does it generate one automatically or something?

Reply:
Yeah, the content scanner generates one at runtime (and the test deliberately invokes the content scanner end-to-end). I specified this filename in the sample config.yaml above.

Reply:
Makes sense. Thanks!